src/sentry/api/serializers/models/project.py

```diff
@@ -115,18 +117,21 @@ def measure_span(op_tag):
                         user=user, project_id__in=project_ids
                     ).values_list("project_id", flat=True)
                 )
-                user_options = {
-                    (u.project_id, u.key): u.value
-                    for u in UserOption.objects.filter(
-                        Q(user=user, project__in=item_list, key="mail:alert")
-                        | Q(user=user, key="subscribe_by_default", project__isnull=True)
+                (
+                    notification_settings_by_project_id,
+                    default_subscribe,
+                ) = transform_to_notification_settings_by_parent_id(
+                    NotificationSetting.objects.get_for_user_by_projects(
+                        ExternalProviders.EMAIL,
+                        NotificationSettingTypes.ISSUE_ALERTS,
+                        user,
+                        item_list,
                     )
-                }
-                default_subscribe = user_options.get("subscribe_by_default", "1") == "1"
+                )
             else:
                 bookmarks = set()
-                user_options = {}
-                default_subscribe = False
+                notification_settings_by_project_id = {}
+                default_subscribe = None
```

Comment:
why is this None rather than False now?

Reply:
Previously `default_subscribe` was the fallback for `"is_subscribed"`. 
```
"is_subscribed": bool(user_options.get((project.id, "mail:alert"), default_subscribe)),
```
I think it made more sense for it to instead be the project-independent notification settings value.
```
is_subscribed = (
	notification_settings_by_project_id.get(project.id, default_subscribe)
    == NotificationSettingOptionValues.ALWAYS
)
```